roles/config-vlans/README.md

```diff
@@ -0,0 +1,93 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with prepping for a vLAN (virtual LAN) install and configuring a vLAN install. VLANs allow for creating virtual area networks to avoid the costly process of buying and configuring new expensive routers for what may be a temporary measure.
```

Comment:
No need to go into the details around what a VLAN is and the benefits of it. I'd keep it simple and short - i.e: something like `This role is used to configure VLANs on a Linux host (RHEL OS family - RHEL, CentOS, Fedora, etc.).`